internal/daemon/daemon.go

```diff
@@ -71,7 +71,7 @@ type Daemon struct {
 func NewDaemon(project string) *Daemon {
 	d := &Daemon{
 		shutdownDoneCh: make(chan error),
-		ReadyChan:      make(chan struct{}),
+		ReadyChan:      make(chan struct{}, 1),
```

Comment:
this change doesnt correlate with the commit message - did you change the wrong channel?

Reply:
Hmm, looks like a rebase artifact or something. Making this channel buffered was entirely unintentional.